src/lib.rs

```diff
@@ -306,6 +306,7 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
     /// assert_eq!(v.get(1), Some(false));
     /// ```
     pub fn push(&mut self, value: bool) {
+        debug_assert_eq!(self.vec.len(), blocks_required::<T>(self.len));
```

Comment:
This assumption is used in several places, but I found out it's not really enforced correctly: `push` broke it. I've added these `debug_assert_eq`s in various places to make sure we catch other cases where it is violated.

Reply:
Good idea.